### PR TITLE
Joe/feat/html login setup

### PR DESCRIPTION
**Pull request by JoeDodgson**

### CODE REVIEW

File: public/stylesheets/less/style.less [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  color: @white;
+}
+
+.break-line {
+  background-color: @primaryColor;
+  height: 1px;
+  width: @half;
+}
+
+.fa-google-plus-g {
```
Comment (Goldenaries658) [inline public/stylesheets/less/style.less:97]: Shouldn't style directly on predefined classes from libraries

File: public/stylesheets/less/style.less [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  color: #ee2b2b;
+  margin-left: 5px;
+  font-size: @fontSizeM;
+}
+
+.email-signin-message {
+  font-size: 0.8em;
```
Comment (Goldenaries658) [inline public/stylesheets/less/style.less:104]: Why does this not use one of the predefined font sizes

File: public/stylesheets/less/style.less [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  background: @tertiaryColor;
+  border-radius: 20px;
+}
+
+.form-container h4 {
+  padding-top: @fontSizeXXXS;
+}
+
+.form-container p {
+  padding-bottom: @fontSizeXXXS;
```
Comment (Goldenaries658) [inline public/stylesheets/less/style.less:133]: Font sizes used to define padding

File: public/stylesheets/less/style.less [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+@green: #016936;
+@teal: #008080;
+@blue: #0e6eb8;
+@violet: #ee82ee;
+@purple: #b413ec;
+@pink: #ff1493;
+@brown: #a52a2a;
+@grey: #a0a0a0;
+@black: #000000;
+@white: white;
```
Comment (Goldenaries658) [inline public/stylesheets/less/style.less:18]:
```suggestion
@white: #eeeeee;
```

File: public/stylesheets/less/style.less [hunk 5]
```diff
@@ -0,0 +1,150 @@
+// Colors
+@primaryColor: #6cad2e;
+@secondaryColor: #444e5d;
+@tertiaryColor: #4e5765;
```
Comment (Goldenaries658) [inline public/stylesheets/less/style.less:4]: Would the tertiary colour not be white?

File: public/login.html [hunk 6]
```diff
@@ -1,45 +1,76 @@
...
+          Or use your email account
+        </div>
+
         <form class="login">
           <div class="form-group">
-            <label for="exampleInputEmail1">Email address</label>
-            <input type="email" class="form-control" id="email-input" placeholder="Email">
+            <input
+              type="email"
+              class="form-control"
```
Comment (Goldenaries658) [inline public/login.html:45]: unused class

File: public/login.html [hunk 7]
```diff
@@ -1,45 +1,76 @@
...
+              id="email-input"
+              placeholder="Enter your email..."
+            />
           </div>
           <div class="form-group">
-            <label for="exampleInputPassword1">Password</label>
-            <input type="password" class="form-control" id="password-input" placeholder="Password">
+            <input
+              type="password"
+              class="form-control"
```
Comment (Goldenaries658) [inline public/login.html:53]: unused class

File: public/login.html (hunk 6 above)
Comment (Goldenaries658) [reply public/login.html:45]: These aren't styled on desktop

File: public/login.html [hunk 8]
```diff
@@ -1,45 +1,76 @@
...
-  </nav>
-  <div class="container">
-    <div class="row">
-      <div class="col-md-6 col-md-offset-3">
-        <h2>Login Form</h2>
+
+      <div class="form-container">
+        <h4>Sign in with Facebook </h4>
+
+        <div class="break-line"></div>
```
Comment (Goldenaries658) [inline public/login.html:35]: What's this doing?

File: public/login.html [hunk 9]
```diff
@@ -1,45 +1,76 @@
...
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.7/lumen/bootstrap.min.css">
-  <link href="stylesheets/style.css" rel="stylesheet">
-</head>
-
-<body>
-  <nav class="navbar navbar-default">
-    <div class="container-fluid">
-      <div class="navbar-header">
+  <body>
+    <main class="center-container">
```
Comment (Goldenaries658) [inline public/login.html:22]: Clean up old semantic classes

File: less-watch-compiler.config.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
+  "allowedExtensions": [".less"],
+  "minified": false,
+  "watchFolder": "public/stylesheets/less",
+  "outputFolder": "public/stylesheets/css"
+}
```
Comment (Goldenaries658) [inline less-watch-compiler.config.json:6]: This doesn't really relate to anything, missing devDependency?

**Comment by JoeDodgson** [comment]
All review comments raised will be addressed in dragos/feat/html-ssearch-setup

**Pull request closed without merging**